application-confluence-migrator-pro-ui/src/main/resources/ConfluenceMigratorPro/Code/MigrationSheet.xml

```diff
@@ -305,18 +313,32 @@
         #if ("$!child.getName()" != "")
          &lt;li class="cfm-doc-tree-leaf"&gt;&lt;a href="$xwiki.getURL($child)"&gt;$nameEscaped&lt;/a&gt;&lt;/li&gt;
         #elseif ($child.containsKey("WebHome") &amp;&amp; $child.size() == 1)
-         &lt;li class="cfm-doc-tree-leaf"&gt;&lt;a href="$xwiki.getURL($child["WebHome"])"&gt;$nameEscaped&lt;/a&gt;&lt;/li&gt;
+         &lt;li class="cfm-doc-tree-leaf"&gt;
+           &lt;a href="$xwiki.getURL($child["WebHome"])"&gt;
+             $nameEscaped
+           &lt;/a&gt;
```

Comment:
Check that this indentation doesn't produce undesirable new spaces (it might be the reason why I didn't indent this)
Same thing for the `#else` branch